### PR TITLE
Vps_solihullsfca

### PR DESCRIPTION
**Pull request by vikramps095**
Vps_solihullsfca

Please Check and review

### CODE REVIEW

**Review by mssquaretech — CHANGES_REQUESTED** [review]
Please remove .idea folder, log.html, screenshots, output.xml, report.html files

**Review by vikramps095 — COMMENTED** [review]
Folder and File Deleted as suggested

**Review by mssquaretech — CHANGES_REQUESTED** [review]
Given statement missing in Feature file. Please make changes and update

**Review by vikramps095 — COMMENTED** [review]
Changes Done

**Review by mssquaretech — APPROVED** [review]

**Comment by mssquaretech** [comment]
This PR is ready to merge.. @vikramps095 
@narendrashakya @Garima2412 @KaranNegi0405 @monalisha-90 . Can anyone of you review and approve to close this PR.
Thanks

File: Vikram/Resource/Pages/Solihullsfc_Homepage.robot [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+
+*** Keywords ***
+Check the Title of the Page
+    ${Title}                    Get Title
+    log to console              Page Title is: ${Title}
+
+Select the Language
+    Click element               ${languageClick}
+    sleep       2s
```
Comment (narendrashakya) [inline Vikram/Resource/Pages/Solihullsfc_Homepage.robot]: Please remove the hard sleep and instead add explicit wait.

**Review by narendrashakya — CHANGES_REQUESTED** [review]

File: Vikram/Resource/Pages/Solihullsfc_Homepage.robot (hunk 1 above)
Comment (vikramps095) [reply Vikram/Resource/Pages/Solihullsfc_Homepage.robot]: Changes Done. Thanks for Quick Review :)

**Review by narendrashakya — APPROVED** [review]
Approved.

File: Vikram/Resource/Pages/Solihullsfc_Homepage.robot (hunk 1 above)
Comment (narendrashakya) [reply Vikram/Resource/Pages/Solihullsfc_Homepage.robot]: Thanks for quick changes.

Comment (vikramps095) [reply Vikram/Resource/Pages/Solihullsfc_Homepage.robot]: Thanks Again for quick reply

Comment (vikramps095) [reply Vikram/Resource/Pages/Solihullsfc_Homepage.robot]: Thanks once Again

**Pull request merged**